### ISSUE TITLE
[ScanFit/SimpleFitGui] fit background only

### ISSUE THREAD

**Issue by maurov** [vasole/pymca#60]
@vasole, unless I'm wrong, in the "simple fit" window (ScanFit) I do not find a way to fit a background only. It would be nice to add in the "function" top down menu, the "None" option. On the other hand, in the "customized fit" window (SimpleFitGui) I do not see the background functions (constant, linear, etc.) in the "background" top down menu list.

Otherwise, is there a simple/quick way of fitting just a linear background without using the console plugin?

Thanks,
Mauro

**Comment by vasole** [comment]
- The only way to just fit a background in the "Simple Fit" is to put such a high number as sensitivity that no peak is found. Alternatively, you can set the constraints of the peak found (if any) to "IGNORE"
- For the customized fit, you can select "User estimated Constant" or "User estimated First Order Polynomial" in the background functions.

**Comment by vasole** [comment]
I agree I should either allow the option None to the simple fit function or to allow the background functions as fitting functions.
